scripts/xtensa-build-zephyr.py

```diff
@@ -483,31 +483,28 @@ def build_platforms():
 		build_cmd = ["west"]
```

Comment:
You added configuration for MTL board, please add missing python dictionary so we can build it too:
	{
		"name": "mtl",
		"PLAT_CONFIG": "intel_adsp_ace15_mtpm",
		"XTENSA_CORE": "ace10_LX7HiFi4_RI_2020_5",
		"XTENSA_TOOLS_VERSION": f"RI-2020.5{xtensa_tools_version_postfix}",
		"RIMAGE_KEY": pathlib.Path("modules", "audio", "sof", "keys", "otc_private_key_3k.pem
	},

Reply:
I will probably remove it and leave it to you to add this alongside other mtl related changes.